### PR TITLE
AF-540: adapted to use appformer generic java model

### PR DESCRIPTION
**Pull request by porcelli**
This is a mandatory step before merging the repos to form AppFormer single repository.

Related PRs:
https://github.com/AppFormer/uberfire/pull/784
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/487
https://github.com/kiegroup/drools/pull/1397
https://github.com/kiegroup/droolsjbpm-integration/pull/1135
https://github.com/kiegroup/guvnor/pull/484
https://github.com/kiegroup/kie-wb-common/pull/1005
https://github.com/kiegroup/drools-wb/pull/560
https://github.com/kiegroup/optaplanner-wb/pull/205

### CODE REVIEW

File: kie-eap-integration/kie-eap-distribution/src/main/filtered-resources/org/uberfire/commons/assembly-component.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!--
+ - Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline kie-eap-integration/kie-eap-distribution/src/main/filtered-resources/org/uberfire/commons/assembly-component.xml]: 2017

File: kie-eap-integration/kie-eap-distribution/src/main/filtered-resources/org/uberfire/project-datamodel-api/assembly-component.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<!--
+ - Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline kie-eap-integration/kie-eap-distribution/src/main/filtered-resources/org/uberfire/project-datamodel-api/assembly-component.xml]: 2017

File: kie-eap-integration/kie-eap-distribution/src/main/filtered-resources/org/uberfire/project-datamodel-commons/assembly-component.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!--
+ - Copyright 2016 Red Hat, Inc. and/or its affiliates.
```
Comment (manstis) [inline kie-eap-integration/kie-eap-distribution/src/main/filtered-resources/org/uberfire/project-datamodel-commons/assembly-component.xml]: 2017

**Comment by porcelli** [comment]
jenkins please test this

File: kie-spring/src/test/java/org/kie/spring/CodeVersion.java [hunk 4]
```diff
@@ -18,6 +18,6 @@
 public class CodeVersion {
 
     // Do not modify this: this is automatically incremented by the maven replacer plugin
-    public final static String VERSION = "7.1.0-SNAPSHOT";
+    public final static String VERSION = "8.0.0-SNAPSHOT";
```
Comment (mariofusco) [inline kie-spring/src/test/java/org/kie/spring/CodeVersion.java]: This String shouldn't be changed.

**Pull request merged**